### ISSUE TITLE
Setting homepage for role does not work

### ISSUE THREAD

**Issue by NxPerc2019** [percussion/percussioncms#448]
This was reproduced in Version 8.0.3 Build 114

When I added SueCushman into a role with Editor being defined as the homepage, it launched into Dashboard and not Editor Module.

Steps to reproduce: 
1)  While logged in as Admin, I created a role called 803Tester.  I set the Homepage to Editor and Assigned SueCushman as the User.
![image-2022-03-02-13-02-07-930](https://user-images.githubusercontent.com/50147765/156421461-910b8238-6c2f-4d2d-8ebc-52c5b8772167.png)

2)  I logged out of CMS as Admin and logged back in as SueCushman.  When I logged in, it took me to Dashboard instead of Editor.
![image-2022-03-02-13-03-09-989](https://user-images.githubusercontent.com/50147765/156421533-29f73208-4e70-48ad-a296-2f04ebabc283.png)

**Comment by natechadwick** [comment]
@NxPerc2019 Was the test user in any other roles or just that role?

**Comment by NxPerc2019** [comment]
The user was in other roles as well.  See screenshot
![usserroles](https://user-images.githubusercontent.com/50147765/156432328-b76b3509-9f06-48fc-9e1f-e1d92f9346a5.png)

**Comment by natechadwick** [comment]
If you remove from all but the role where you changed the home page - does the home page work as expected?  I am thinking it is just grabbing the first or last roles settings for that.

**Comment by NxPerc2019** [comment]
You are correct - I removed all other roles and it launched into the Editor page as it should have.  I guess a documentation update may be in order to understand which role gets the precedence.  Thanks Nate!

**Comment by natechadwick-intsof** [comment]
Issue migrated to: https://github.com/intersoftdatalabs-in/percussioncms/issues/959

**Comment by natechadwick-intsof** [comment]
Issue closed and migrated to: https://github.com/intersoftdatalabs-in/percussioncms/issues/959
